src/details.js

```diff
@@ -28,43 +28,45 @@ let redact;
 
 module.exports = function details() {
   const Parser = this.Parser;
-  const tokenizers = Parser.prototype.blockTokenizers;
-  const methods = Parser.prototype.blockMethods;
-  const restorationMethods = Parser.prototype.restorationMethods;
-
-  if (restorationMethods) {
-    restorationMethods.details = function(add, node, content, children) {
-      const colonCount = node.redactionData;
-      const open = add({
-        type: 'paragraph',
-        children: [
-          {
-            type: 'rawtext',
-            value: `${colon.repeat(colonCount)} details [${content}]`
-          }
-        ]
-      });
-
-      const childNodes = children.map(child => add(child));
-
-      const close = add({
-        type: 'paragraph',
-        children: [
-          {
-            type: 'rawtext',
-            value: colon.repeat(colonCount)
-          }
-        ]
-      });
-
-      return [open, ...childNodes, close];
-    };
+  if (Parser) {
```

Comment:
This needed to be added to allow tests to work. The way plugins are passed into the tests doesn't differentiate between parser and compiler plugins. This could be changed at some point but this change is already large enough.